lib/runner/job.ex

```diff
@@ -63,12 +55,11 @@ defmodule ElixirBench.Runner.Job do
     end
   end
 
-  defp ensure_no_other_jobs! do
+  def ensure_no_other_jobs! do
     %{active: 0} = Supervisor.count_children(ElixirBench.Runner.JobsSupervisor)
```

Comment:
I still think we can easily keep this private and test as part of the `start_job` logic as it raises in the error case so no other logic is actually triggered.